pkg/systemd/services/services.jsx

```diff
@@ -729,9 +729,17 @@ class ServicesPageBody extends React.Component {
             const unit_path = get_unit_path(unit_id);
 
             if (unit_path === undefined) {
-                return <EmptyStatePanel
+                return (<EmptyStatePanel
                             icon={ExclamationCircleIcon}
-                            title={_("Unit not found")} />;
+                            title={_("Unit not found")}
+                            secondary={
```

Comment:
Using `secondary` seems like the most logical solution. However it feels like there is too much space now.
`secondary`:
![Screenshot from 2022-02-11 07-29-24](https://user-images.githubusercontent.com/12330670/153546929-cfa8e505-2b5d-4fb7-bcf9-5657e4f51273.png)

`paragraph`:
![paragraph](https://user-images.githubusercontent.com/12330670/153546938-82625b5f-bbdd-45c5-bffa-c0232b9dba6e.png)

WDYT?